docs/index.md

```diff
@@ -35,6 +35,57 @@ non-public development, custom hardware handled by vendor
   source hardware and open instruction set architecture. Feel free to check the
   website about recent vPub plans or subscribe to newsletter.
 
+## Newcommers
```

Comment:
```suggestion
## Newcomers
```

Reply:
I wonder if this shouldn't go to separate document. I guess this section may grow over time.

Reply:
Let's leave it here for now, at the end we can decide if it's big enough to get a separate section.